### PR TITLE
PMM-8986 Fix checks timeouts

### PR DESCRIPTION
**Pull request by artemgavrilov**
https://jira.percona.com/browse/PMM-8986

- [ ] Build: https://github.com/Percona-Lab/pmm-submodules/pull/2282

- [ ] Links to other linked pull requests (optional).
---
- [ ] Tests passed.
- [ ] Feature build pass.
- [ ] (Re)requested review.
- [ ] Fix conflicts with target branch.
- [ ] Update API dependency.
- [ ] Update jira ticket description if needed.
- [ ] Attach screenshots/console output to confirm new behavior to jira ticket, if applicable.

### CODE REVIEW

**Review by PavelKhripkov — APPROVED** [review]

File: services/checks/checks.go [hunk 1]
```diff
@@ -56,9 +56,11 @@ const (
 	envResendInterval    = "PERCONA_TEST_CHECKS_RESEND_INTERVAL"
 	envDisableStartDelay = "PERCONA_TEST_CHECKS_DISABLE_START_DELAY"
 
-	checksTimeout       = 5 * time.Minute  // timeout for checks downloading/execution
-	resultTimeout       = 20 * time.Second // should greater than agents.defaultQueryActionTimeout
-	resultCheckInterval = time.Second
+	checkExecutionTimeout  = 5 * time.Minute
+	platformRequestTimeout = 2 * time.Minute
+	resultAwaitTimeout     = 20 * time.Second // should greater than agents.defaultQueryActionTimeout
```
Comment (BupycHuk) [inline services/checks/checks.go:61]:
```suggestion
	resultAwaitTimeout     = 20 * time.Second // should be greater than agents.defaultQueryActionTimeout
```

**Review by BupycHuk — APPROVED** [review]

File: api-tests/server/checks_test.go [hunk 2]
```diff
@@ -298,3 +216,22 @@ func TestChangeSecurityChecks(t *testing.T) {
 		})
 	})
 }
+
+func enableSTT(t *testing.T, enable bool) {
```
[the remark this answers is not in the document]
Comment (PavelKhripkov) [reply api-tests/server/checks_test.go]: Changed to toggleSTT

File: services/checks/checks.go (hunk 1 above)
Comment (PavelKhripkov) [reply services/checks/checks.go]: Done

File: services/checks/checks.go [hunk 3]
```diff
@@ -56,9 +56,11 @@ const (
 	envResendInterval    = "PERCONA_TEST_CHECKS_RESEND_INTERVAL"
 	envDisableStartDelay = "PERCONA_TEST_CHECKS_DISABLE_START_DELAY"
 
-	checksTimeout       = 5 * time.Minute  // timeout for checks downloading/execution
-	resultTimeout       = 20 * time.Second // should greater than agents.defaultQueryActionTimeout
-	resultCheckInterval = time.Second
+	checkExecutionTimeout  = 5 * time.Minute
+	platformRequestTimeout = 2 * time.Minute
```
[the remark this answers is not in the document]
Comment (PavelKhripkov) [reply services/checks/checks.go]: Done

File: api-tests/server/checks_test.go [hunk 4]
```diff
@@ -32,21 +33,10 @@ import (
...
-		res, err := client.ChangeSettings(&server.ChangeSettingsParams{
-			Body: server.ChangeSettingsBody{
-				EnableStt:       true,
-				EnableTelemetry: true,
-			},
-			Context: pmmapitests.Context,
-		})
-		require.NoError(t, err)
-		assert.True(t, res.Payload.Settings.SttEnabled)
-		assert.True(t, res.Payload.Settings.TelemetryEnabled)
```
[the remark this answers is not in the document]
Comment (PavelKhripkov) [reply api-tests/server/checks_test.go:49]: Its done in https://github.com/percona/pmm-managed/blob/main/api-tests/server/settings_test.go

**Pull request merged**